core/idl/preferences/preferences.djinni

```diff
@@ -74,3 +76,58 @@ PreferencesEditor = interface +c {
     # Clear all preferences.
     clear();
 }
+
+PreferencesChangeType = enum {
+    PUT_TYPE;
+    DELETE_TYPE;
```

Comment:
The type suffix is redundant with the enumeration name
```suggestion
    PUT;
    DELETE;
```

Reply:
I'll do this